src/Sarif/Writers/PrereleaseCompatibilityTransformer.cs

```diff
@@ -409,90 +409,58 @@ private static void RemoveToolLanguage(JObject run)
 
         private static void UpdateAllReportingDescriptorPropertyTypes(JObject run)
         {
-            // Access and modify run.tool
-            JObject tool = (JObject)run["tool"];
-            UpdateToolReportingDescriptorPropertyTypes(tool);
-
-            // Access and modify run.conversion.tool
-            if (run["conversion"] is JObject conversion)
+            string[] reportingDescriptorPathsToUpdate =
```

Comment:
>reportingDescriptorPathsToUpdate [](start = 21, length = 32)

note: i picked up only a simple method for now and converted it to demonstrate usage (and get tests to do their job).